src/libostree/ostree-impl-system-generator.c

```diff
@@ -126,34 +126,6 @@ require_internal_units (const char *normal_dir, const char *early_dir, const cha
 #endif
 }
 
-// Resolve symlink to return osname
-static gboolean
```

Comment:
How is the Android Boot implementation gonna work if we remove this?

Reply:
I'm a fan of this feature, I'm not convinced if we tested this with the aboot feature it would work though.

Maybe it's just the sake of adding a configuration to opt-in to use this feature and not delete the old way.